bin/Powheg/run_pwg.py

```diff
@@ -531,6 +532,13 @@ def runGetSource(parstage, xgrid, folderName, powInputName, process, noPdfCheck,
   
 echo "ANALYSIS=none " >> tmpfile
 
+# Add libraries now
+NEWRPATH1=`ls /cvmfs/cms.cern.ch/${SCRAM_ARCH}/external/gcc/*/* | grep "/lib64" | head -n 1`
+NEWRPATH1=${NEWRPATH1%?}
+NEWRPATH2=`ls /cvmfs/cms.cern.ch/${SCRAM_ARCH}/external/zlib-x86_64/*/* | grep "/lib" | head -n 1`
+NEWRPATH2=${NEWRPATH2%?}
+echo "RPATHLIBS= -Wl,-rpath,${NEWRPATH1} -L${NEWRPATH1} -lgfortran -lstdc++ -Wl,-rpath,${NEWRPATH2} -L${NEWRPATH2} -lz" >> tmpfile
+
```

Comment:
@covarell , why not add lhapdf libaray path in RPATHLIBS?

Reply:
This is already the default in all POWHEG Makefiles, so no need to change